spec.bs

```diff
@@ -704,6 +704,7 @@ dictionary AuctionAdConfig {
   USVString trustedScoringSignalsURL;
   sequence<USVString> interestGroupBuyers;
   Promise<any> auctionSignals;
+  record<USVString, DOMString> requestedSize;

```

Comment:
What values are meant to be stuffed here? I just want to make sure I understand the need for USVString?

Reply:
Here's a four example values that could be passed to `requestedSize`:

1. `{width: '100', height: '100'}`
2. `{width: '100px', height: '200px'}`
3. `{width: '75sw', height: '25sh'}`
4. `{width: '100', height: '25sh'}`

Should I instead make a separate dictionary? e.g.
```
dictionary AuctionAdSize {
  required USVString width;
  required USVString height;
};
```

Reply:
I think a record is fine. But the key and value types should both be DOMString in this case